app/dashboard/static/js/app/view-tests-all.2020.5.js

```diff
@@ -43,62 +43,147 @@ require([
+            for (key in gTestCount) {
+                if (gTestCount.hasOwnProperty(key)) {
+                    updateOrStageCount(key, gTestCount[key]);
+                }
+            }
+        }
+    }
+
+    function getBatchTestCountDone(results) {
+        var batchData;
+        batchData = results[0].result;
+
         function parseBatchData(data) {
-            gTestStatus[data.operation_id] = (data.result[0].count == 0 ? "PASS" : "FAIL");
-            var node = document.getElementById(data.operation_id);
-            if (node != null){
-                node.firstChild.replaceWith(ttest.statusNode(gTestStatus[data.operation_id]));
+            updateOrStageCount(
+                data.operation_id, parseInt(data.result[0].count, 10));
+        }
+
+        if (batchData.length > 0) {
+            batchData.forEach(parseBatchData);
+            if (!gDrawEventBound) {
+                gDrawEventBound = true;
+                gTestsTable.addDrawEvent(updateTestTable);
             }
         }
-        results.result.forEach(parseBatchData);
     }
 
-    function getBatchStatusFailed() {
-        console.log("getBatchStatusFailed()");
+    function getBatchTestCountFailed() {
+        console.log("getBatchTestCountFailed()");
     }
 
-    function getBatchStatus(results) {
+    function getBatchTestCount(results) {
         var batchOps;
         var deferred;
         function createBatchOp(result) {
             var qStr;
-            var plan = result.name;
+            var qHead;
+            var opId;
+            var opIdTail = result._id.$oid;
 
             qStr = URI.buildQuery({
                 'kernel': result.kernel,
-                'plan': plan,
+                'plan': result.name,
             });
 
-            /* Number of test case regressions */
+            // Get total tests count.
+            opId = 'test-total-count-';
+            opId += opIdTail;
             batchOps.push({
                 method: 'GET',
-                operation_id: 'status-' + result._id.$oid,
+                operation_id: opId,
+                resource: 'count',
+                document: 'test_case',
+                query: qStr
+            });
+
+            // Get successful tests count.
+            opId = 'test-success-count-';
+            opId += opIdTail;
+            qHead = 'status=PASS&';
+            qHead += qStr;
+            batchOps.push({
+                method: 'GET',
+                operation_id: opId,
+                resource: 'count',
+                document: 'test_case',
+                query: qHead
+            });
+
+            // Get regressions count.
+            opId = 'test-fail-count-';
+            opId += opIdTail;
+            batchOps.push({
+                method: 'GET',
+                operation_id: opId,
                 resource: 'count',
                 document: 'test_regression',
-                query: qStr,
+                query: qStr
```

Comment:
I actually think it's better to keep the trailing comma, because if an argument gets added later this line doesn't need to be edited to add a comma.

JSON syntax requires the last item to not have a trailing comma, but this is a Javascript dictionary.

So it's OK either way, but not worth removing the comma imho.